docs/execution_providers/TensorRT-ExecutionProvider.md

```diff
@@ -67,9 +67,13 @@ ORT_TENSORRT_MIN_SUBGRAPH_SIZE: minimum node size in a subgraph after partitioni
 
 ORT_TENSORRT_FP16_ENABLE: Enable FP16 mode in TensorRT
 
-By default TensorRT execution provider builds an ICudaEngine with max workspace size = 1 GB, max partition iterations = 1000, min subgraph size = 1 and FP16 mode is disabled.
+ORT_TENSORRT_ENGINE_CACHE_ENABLE: Enable TensorRT engine caching
```

Comment:
I think we need some more documentation on the engine caching. (why it needed, how does it work,
when would you use it, and what are some of the pitfalls and limitations.
examples might be 
-if you enabled fp16 and serialized engines, you need to enable fp16 when deploying/running it. 
-engines are built specifically for the underlying hardware and aren't portable. 
-caveats about input shape changes.

Reply:
Good point. I've added more explanations in the doc.